app.js

```diff
@@ -29,3 +32,48 @@ var layer = L.tileLayer(tiles, {
 
 // add the tile layer to the map
 layer.addTo(map);
+
+//// 2nd Milestone - POKEMON PAST LOCATION
+//// 3rd Milestone - POKEMON INFO
+
+// custom icons
+var LeafIcon = L.Icon.extend({
+    options: {
+        iconSize:     [50, 46],
+        popupAnchor:  [-7, -20]
+    }
+});
+
+// mark Pokemons
+L.marker([48.262299, 11.669776], {icon: new LeafIcon({iconUrl: 'map/css/images/pokemon/pikachu.gif'})})
+    .addTo(map).bindPopup('Pokemon: <b>Pikachu</b><br/> <br/>' +
```

Comment:
I suggest you to create a pop-up object, then a marker, and finally bind the pop-up to the marker :)